wandb/integration/fastai/__init__.py

```diff
@@ -41,10 +41,16 @@
 from pathlib import Path
 import random
 
+if wandb.TYPE_CHECKING:
+    from ...sdk.lib import ipython
+else:
+    from ...sdk_py27.lib import ipython
+
 try:
     import matplotlib
 
-    matplotlib.use("Agg")  # non-interactive backend (avoid tkinter issues)
+    if ipython._get_python_type() != "jupyter":
```

Comment:
I think this is less ugly:
wandb.wandb_lib.ipython._get_python_type()

Reply:
oh nice, changed.